lib/dex-oracle/driver.rb

```diff
@@ -47,38 +47,18 @@ def install(dex)
       raise "#{Resources.dx} does not exist and is required for DexMerger" unless File.exist?(Resources.dx)
       raise "#{Resources.driver_dex} does not exist" unless File.exist?(Resources.driver_dex)
       logger.debug("Merging input DEX (#{dex.path}) and driver DEX (#{Resources.driver_dex}) ...")
-      tf = Tempfile.new(%w(oracle-driver .dex))
-      cmd = "java -cp #{Resources.dx} com.android.dx.merge.DexMerger #{tf.path} #{dex.path} #{Resources.driver_dex}"
-      stdout, stderr = exec(cmd)
-      # Ideally, it says something like this:
-      # Merged dex #1 (36 defs/87.9KiB)
-      # Merged dex #2 (1225 defs/1092.7KiB)
-      # Result is 1261 defs/1375.0KiB. Took 0.2s
-      # But it might say this:
-      # Exception in thread "main" com.android.dex.DexIndexOverflowException: Cannot merge new index 65776 into a non-jumbo instruction!
-      if stderr.start_with?('Exception in thread "main"')
-        logger.error("Failure to merge input DEX and driver DEX:\n#{stderr}")
-        if stderr.include?('DexIndexOverflowException')
-          logger.error("Your input DEX inexplicably contains const-string and const-string/jumbo. This probably means someone fucked with it. In any case, it means DexMerge is failing because there are too many strings.\nTry this: baksmali the DEX, replace all const-string instructions with const-string/jumbo, then recompile with smali and use that DEX as input. Sorry, I don't want to do this for you. It's too complicated.")
-        end
-        exit -1
-      end
-      tf.close
-
```

Comment:
WHYTHISWORKS: This produced the following errors:
`[ERROR] 2017-08-10 23:06:38: Failure to merge input DEX and driver DEX:
Exception in thread "main" com.android.dex.DexIndexOverflowException: method ID not in [0, 0xffff]: 65536
	at com.android.dx.merge.DexMerger$6.updateIndex(DexMerger.java:495)
	at com.android.dx.merge.DexMerger$IdMerger.mergeSorted(DexMerger.java:261)
	at com.android.dx.merge.DexMerger.mergeMethodIds(DexMerger.java:504)
	at com.android.dx.merge.DexMerger.mergeDexes(DexMerger.java:161)
	at com.android.dx.merge.DexMerger.merge(DexMerger.java:188)
	at com.android.dx.merge.DexMerger.main(DexMerger.java:1120)


[ERROR] 2017-08-10 23:06:38: Your input DEX inexplicably contains const-string and const-string/jumbo. This probably means someone fucked with it. In any case, it means DexMerge is failing because there are too many strings.
Try this: baksmali the DEX, replace all const-string instructions with const-string/jumbo, then recompile with smali and use that DEX as input. Sorry, I don't want to do this for you. It's too complicated.`

The second one is simply an invalid one and I think the first one is because of too many methods but I'm not sure